_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+				<PlanPriceDisplay
+					dailyPrice={ sitePlans[ 'daily-backup' ].price.yearly.text }
+					yearlyPrice={ sitePlans[ 'realtime-backup' ].price.yearly.amount }
+				/>
+			) }
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
+	const { dailyPrice, yearlyPrice } = props;
+	const perYearPriceRange = `${ dailyPrice }-${ yearlyPrice } /year`;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<SlashedPrice />
+			<div className="plans-price__container">
+				<span className="plans-price__span">{ perYearPriceRange }</span>
+			</div>
+		</div>
+	);
+}
+
+function SlashedPrice() {
+	return (
+		<div className="slashed-price__container" style={ { marginRight: '14px' } }>
+			<div className="slashed-price__slash"></div>
+			{ /* TODO: get this from an API or calculate, currently unsure how to get this increased price */ }
+			<div className="slashed-price__price">{ '$15-25' }</div>
+		</div>
+	);
+}
+
+function SingleProductBackupBody( props ) {
+	const { selectedBackupType, setSelectedBackupType, sitePlans } = props;
+
+	const upgradeLinks = {
+		'real-time': 'https://wordpress.com/jetpack/connect/pro',
+		daily: 'https://wordpress.com/jetpack/connect/premium',
+	};
+
+	const upgradeTitles = {
+		'real-time': __( 'Upgrade to Real-Time Backups' ),
+		daily: 'Upgrade to Daily Backups',
+	};
+
+	function handleSelectedBackupTypeChange( event ) {
+		setSelectedBackupType( event.target.value );
+	}
+
+	return (
+		<div className="plans-section__body">
+			<p>
+				{ __(
+					'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{ext/}}{{/a}}',
+					{
+						components: {
+							a: <a href="https://jetpack.com/upgrade/backup/" />,
+							ext: (
+								<>
+									<span>
+										<Gridicon icon="external" size="12" />
+									</span>
+								</>
+							),
+						},
+					} // TODO: make icon color change with link
+				) }
+			</p>
+
+			<h4>Backup options:</h4>
+			<div
+				style={ {
+					display: 'flex',
+					flexDirection: 'row',
+					justifyContent: 'center',
+				} }
+			>
+				<PlanRadioButton
+					planName={ __( 'Daily Backups' ) }
+					radioValue={ 'daily' }
+					planPrice={ sitePlans && sitePlans[ 'daily-backup' ].price.yearly.text }
+					checked={ 'daily' === selectedBackupType }
+					onChange={ handleSelectedBackupTypeChange }
+				/>
+				<PlanRadioButton
+					planName={ __( 'Real-Time Backups' ) }
+					radioValue={ 'real-time' }
+					planPrice={ sitePlans && sitePlans[ 'realtime-backup' ].price.yearly.text }
+					checked={ 'real-time' === selectedBackupType }
+					onChange={ handleSelectedBackupTypeChange }
+				/>
+			</div>
+
+			<div style={ { textAlign: 'center', marginTop: '23px', marginBottom: '10px' } }>
+				<a href={ upgradeLinks[ selectedBackupType ] } type="button" class="dops-button is-primary">
```

Comment:
I was wanting to use `a` instead of `button` for the semantics but I see that our `Button` component handles that internally so I've switched it, good call!